dashboard/v1.1/src/pages/Monitoring/RouteDetails.tsx

```diff
@@ -55,6 +54,8 @@ const format = (data: RouteDetails) => {
   const pingMax: chartData[] = [];
   const jitter: chartData[] = [];
   const name = data.name;
+  const fetchTime = data.fetchTime;
+  const queryTime: number[] = [0, 0, 0];
```

Comment:
Can we not have this as an object? using an array for this purpose is weird.

Reply:
My bad, an object struct will be a better fit for the use case. I will do the changes.

Reply:
Please add a comment on why `queryTime` has a structure as you have mentioned (e.g. ping, jitter, etc.)